packages/devkit/devkit.1.3/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "igor@ahrefs.com"
+authors: "Ahrefs <github@ahrefs.com>"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/devkit"
+dev-repo: "git://github.com/ahrefs/devkit.git"
+bug-reports: "https://github.com/ahrefs/devkit/issues"
+synopsis: "Development kit - general purpose library"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc"{with-doc}]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs "@install" "@runtest"{with-test} "@doc"{with-doc}]
```
for the record you can also use the same trick with tests

Reply:
thanks, didn't know about `@runtest`